### ISSUE TITLE
drawImage does implicit conversion of image position to integer

### ISSUE THREAD

**Issue by bchevalier** [Automattic/node-canvas#395]
When trying to draw a canvas into another canvas there seems to be an implicit conversion of the drawing position into integer values. In the following code, my imageCanvas is drawn at position (x, y) = (9, 9) instead of  (x, y) = (9.99, 9.99).

``` javascript
        var canvas = new Canvas();
        var context = canvas.getContext('2d');

        canvas.width = 100;
        canvas.height = 100;

        context.clearRect(0, 0, canvas.width, canvas.height);

        context.save();
        context.transform(1, 0, 0, 1, 0, 0);
        context.drawImage(imageCanvas, 9.99, 9.99);
        context.restore();
```

In addition, this integer conversion propagates, if I scale it by a factor of 10. The position of the image becomes  (x, y) = (90, 90) instead of (99.9, 99.9).

By the way, the problem does not seem to exist when replacing drawImage by fillRect(9.99, 9.99, rectWidth, rectHeight), which makes me thing that the behavior of the drawImage is wrong.

**Comment by LinusU** [comment]
Could anyone confirm that this is working on Safari, Chrome or Firefox?

**Comment by zbjornson** [comment]
This seems to have been fixed in 1.2.0 (36bd6498). Chrome and node-canvas look comparable now.
